src/js/node/stream/Writable.hx

```diff
@@ -147,7 +147,13 @@ extern class Writable<TSelf:Writable<TSelf>> extends Stream<TSelf> implements IW
 typedef WritableNewOptions = {
 	@:optional var highWaterMark:Int;
 	@:optional var decodeStrings:Bool;
+	@:optional var defaultEncoding:String;
 	@:optional var objectMode:Bool;
+	@:optional var emitClose:Bool;
+	@:optional var write:Dynamic->String->(Null<Error>->Void)->Void;
+	@:optional var writev:Array<{chunk:Dynamic,encoding:String}>->(Null<Error>->Void)->Void;
+	@:optional var destroy:Null<Error>->(Null<Error>->Void)->Void;
+	@:optional var autoDestroy:Bool; 
```

Comment:
I'm a bit confused where `defaultEncoding` and `autoDestroy` come from, the interface [you linked](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ee9bf2dc97503cd297058bfe8d69477bca89a4a1/types/node/v10/stream.d.ts#L98-L106) doesn't seem to have those?

Reply:
Thank you very much! This is a careless mistake.

`autoDestroy` has been added since v11. v12 has not become LTS yet, so I removed it with this commit.

Reply:
Ah, I see. Not sure what our exact policy is regarding LTS versions and such... In fact, the version number is still at 6.9.x. @nadako?

Reply:
I created an issue #127